graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetConfig.java

```diff
@@ -58,7 +58,9 @@
         @JsonProperty("messages")
         MESSAGES,
         @JsonProperty("events")
-        EVENTS
+        EVENTS,
+        @JsonProperty("generic")
+        GENERIC
```

Comment:
What's the reasoning behind calling this `generic`? Wouldn't it be better to call this `gim` or `graylog-schema` or something similar?

And should we include a version number? Or how would we handle different versions of the schema?

Reply:
We had mentioned on the side that we would call the type "generic" in our last meeting about it, at least that is what I had written down. I cannot recall the reasoning though. So, I would rename it to `gim` - and include the versioning, so it would be `gim-v1` for now.